src/application.c

```diff
@@ -23,7 +23,6 @@ void Run() {
   SetUserPreference("environment", 0.30);
   SetUserPreference("health", 0.40);
   // NOLINTEND(cppcoreguidelines-avoid-magic-numbers,readability-magic-numbers)
-
```

Comment:
@SebastianMygind Why?

Reply:
a mistake. If i am understanding correctly that I removed a newline

Reply:
Yeah, you did, please bring it back